### PR TITLE
feat: journey view social image

### PR DESCRIPTION
**Pull request by GeronimoJohn**
# Description

Created the social image component. Added this component to the journey view component.

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/29147486/todos/5294620579)

# How should this PR be QA Tested?

Within Storybook
- [ ] Users should be able to see the social share image of a journey

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged into main

### CODE REVIEW

File: apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            alt={journey.primaryImageBlock?.alt}
+            style={{
+              display: 'flex',
+              justifyContent: 'center',
+              alignItems: 'center',
+              width: 213,
+              height: 167,
+              objectFit: 'cover',
+              borderRadius: 12,
+              marginRight: smUp ? 0 : 'auto',
```
Comment (storyworks) [inline apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.tsx:29]: https://mui.com/material-ui/customization/breakpoints/#css-media-queries 

Use this method over useMediaQuery. useMediaQuery have server side rendering limitations

File: apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+const Template: Story = ({ ...args }) => (
+  <MockedProvider>
+    <JourneyProvider value={{ journey: args.journey }}>
+      <SocialImage />
+    </JourneyProvider>
+  </MockedProvider>
+)
+
+export const Default = Template.bind({})
```
Comment (storyworks) [inline apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.stories.tsx:22]: Add placeholder and loading stories?

File: apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import { render } from '@testing-library/react'
+import { JourneyProvider } from '@core/journeys/ui/JourneyProvider'
+import { publishedJourney } from '../data'
+import { SocialImage } from './SocialImage'
+
+jest.mock('@mui/material/useMediaQuery', () => ({
```
Comment (storyworks) [inline apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.spec.tsx:6]: Won't need this after removing useMediaQuery

File: apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.tsx (hunk 1 above)
Comment (GeronimoJohn) [reply apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.tsx]: Ended up using flexbox to center the component. More design updates will be added on once the templates page PR is merged

File: apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        ) : (
+          <Box
+            style={{
+              display: 'flex',
+              justifyContent: 'center',
+              alignItems: 'center',
+              borderRadius: 12,
+              width: 213,
+              height: 167,
+              backgroundColor: '#EFEFEF'
```
Comment (storyworks) [inline apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.tsx:43]: Use a token instead of hard coding color

Comment (GeronimoJohn) [reply apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.tsx:43]: I have tried using this `backgroundColor: 'background.default'` and it didn't really work for me

File: apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.tsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    <Box
+      style={{
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center'
+      }}
+    >
+      {journey != null ? (
+        journey.primaryImageBlock?.src != null ? (
+          <Box
```
Comment (storyworks) [inline apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.tsx:20]: Use NextImage

File: apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.tsx (hunk 4 above)
Comment (storyworks) [reply apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.tsx:43]: Use `sx={{...}}` `style` doesn't recognize Mui tokens

File: apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.tsx [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+import { ReactElement } from 'react'
+import ImageIcon from '@mui/icons-material/Image'
+import Skeleton from '@mui/material/Skeleton'
+
+export function SocialImage(): ReactElement {
+  const { journey } = useJourney()
+
+  return (
+    <Box
+      style={{
```
Comment (storyworks) [inline apps/journeys-admin/src/components/JourneyView/SocialImage/SocialImage.tsx:12]: Same here, don't use `style` use `sx` just as general practice

**Review by storyworks — APPROVED** [review]

**Pull request merged**